### PR TITLE
[MSYS-715]Setup appveyor 

### PR DESCRIPTION
**Pull request by dheerajd-msys**
Added `appveyor.yml` config file to run unit tests.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - ruby_version: "24-x64"
+    - ruby_version: "24"
+
+clone_folder: c:\projects\win32-certstore
+clone_depth: 1
+skip_tags: true
+branches:
+  only:
+    - master
+    - chef-12
```
Comment (btm) [inline appveyor.yml]: copy-paste cruft?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - echo %PATH%
+  - ps: $env:OMNIBUS_RUBYGEMS=$(findstr rubygems omnibus_overrides.rb | %{ $_.split(" ")[3] })
+  - echo %OMNIBUS_RUBYGEMS%
+  - ps: $env:OMNIBUS_BUNDLER=$(findstr bundler omnibus_overrides.rb | %{ $_.split(" ")[3] })
+  - echo %OMNIBUS_BUNDLER%
+  - ruby --version
+  - gem --version
+  - bundler --version
+  - gem uninstall bundler -a -x
```
Comment (btm) [inline appveyor.yml]: I'm looking into some failures in chef/chef on appveyor that make me suspect we need to not try to uninstall bundler. See https://github.com/chef/chef/pull/6597.

Comment (dheerajd-msys) [reply appveyor.yml]: Thanks @btm, I will remove this line.

File: appveyor.yml (hunk 1 above)
Comment (dheerajd-msys) [reply appveyor.yml]: My bad, I will remove this.

**Pull request merged**